### DISCUSSION TITLE
Upgrade Button Improvements

### DISCUSSION THREAD

**Discussion by marwie** [coollabsio/coolify#2203] [Feature Requests]
Hey, instead of just showing "Upgrade" it would be great to see more information there - like which version I'm currently on and which version I would upgrade to - ideally with a changelog or link to the github release page. 

As far as I can tell there's also no trivial way to downgrade again?

Additionally I think it would be good to be asked again before an update is performed. Right now clicking "Upgrade" will run immediately without any apparent feedback that the update is running 

![image](https://github.com/coollabsio/coolify/assets/5083203/68382baa-d3df-4f0c-bf7f-0980f37c7633)

Originally posted here: https://discord.com/channels/459365938081431553/1239208858862489630/1239876347598475284

**Comment by RayBB** [comment]
I also think it's a great suggestion.
It will also make coolify a bit more professional and appealing to the more cautious audience that might be evaluating coolify but concerned about breaking changes.

Related to: https://github.com/coollabsio/coolify/discussions/2136
And possibly related to: https://github.com/coollabsio/coolify/discussions/1652

**Comment by andrasbacsai** [comment]
I have improved the upgrade process by displaying a confirmation, providing a link to the changelog, and showing the exact progress. This will be added in v281.

**Reply by marwie** [reply]
Thank you, that's great. Do you also have plans to allow easy downgrading or would you say this is just by restoring a coolify backup?

**Reply by andrasbacsai** [reply]
There is a [downgrade process](https://coolify.io/docs/downgrade), but I do not prefer to have this on the UI. It should be a manual effort.
